cmplibrary/src/main/java/com/sourcepoint/gdpr_cmplibrary/GDPRUserConsent.java

```diff
@@ -41,4 +41,15 @@ public GDPRUserConsent(JSONObject jConsent) throws JSONException, ConsentLibExce
     public JSONObject getJsonConsents() {
         return jsonConsents;
     }
+
+    private static JSONObject getEmptyUserConsents() throws JSONException{
```

Comment:
Do we really need a static method here? It should be avoided whenever possible and I think this is the case here.

Reply:
yes we can remove static method from here and initialization logic in constructor itself